sky/data/storage.py

```diff
@@ -532,12 +571,13 @@ def _validate_source(
           source: str; File path where the data is initially stored. Can be a
             local path or a cloud URI (s3://, gs://, r2:// etc.).
             Local paths do not need to be absolute.
-          mode: StorageMode; StorageMode of the storage object
+          mode: storage_utils.StorageMode; StorageMode of the storage object
 
         Returns:
           Tuple[source, is_local_source]
           source: str; The source path.
-          is_local_path: bool; Whether the source is a local path. False if URI.
+          is_local_path: bool; Whether the source is a local path.
+            False if URI.
```

Comment:
```suggestion
          is_local_path: bool; Whether the source is a local path. False if URI.
```
Why are we changing this? Seems like a remnant

Reply:
Keeping those two lines on a single line becomes longer than col 80.